File: Python/0002.py
times = []

for _ in range(0, 3):
    times.append(int(input()))

ords = times.copy()
ords.sort()

for i in range(0, 3):
    for j in range(0, 3):
        if times[j] == ords[i]:
            print(j + 1)
            break

File: C++/0002.cpp
#include <algorithm>
#include <iostream>

int main() {
    int times[3], ord[3];
    for (int i = 0; i < 3; ++i) {
        std::cin >> times[i];
    }

    std::copy(std::begin(times), std::end(times), std::begin(ord));

    std::sort(std::begin(ord), std::end(ord));

    for (int i = 0; i < 3; ++i) {
        for (int j = 0; j < 3; ++j) {
            if (times[j] == ord[i]) {
                std::cout << j + 1 << "\n";
                break;
            }
        }
    }

    return 0;
}